### PR TITLE
test: deno fly.io deployment test

### PR DESCRIPTION
**Pull request by jacob-ebey**
Closes: #

- [ ] Docs
- [ ] Tests

### CODE REVIEW

File: packages/create-remix/templates/deno/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+WORKDIR /app/
+
+COPY --from=build /app/build /app/build
+COPY --from=build /app/public /app/public
+
+RUN deno cache ./build/index.js
+
+EXPOSE 8080
+
+CMD ["run", "--allow-net", "--allow-read", "--allow-env", "./build/index.js"]
```
Comment (mcansh) [inline packages/create-remix/templates/deno/Dockerfile:39]: should this be `deno` prefixed?

```suggestion
CMD ["deno", "run", "--allow-net", "--allow-read", "--allow-env", "./build/index.js"]
```

File: scripts/deployment-test/fly-deno.mjs [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import path from "path";
+import { spawnSync } from "child_process";
+import fse from "fs-extra";
+import toml from "@iarna/toml";
+
+import { sha, getSpawnOpts, runCypress, addCypress } from "./_shared.mjs";
+import { createApp } from "../../build/node_modules/create-remix/index.js";
+
+let APP_NAME = `remix-fly-${sha}`;
```
Comment (mcansh) [inline scripts/deployment-test/fly-deno.mjs:9]:
```suggestion
let APP_NAME = `remix-fly-deno-${sha}`;
```

**Review by mcansh — CHANGES_REQUESTED** [review]
looks great besides two teeny tiny things

File: packages/create-remix/templates/deno/Dockerfile (hunk 1 above)
Comment (jacob-ebey) [reply packages/create-remix/templates/deno/Dockerfile:39]: The default command is "deno" for this image so it's not needed :)

Comment (mcansh) [reply packages/create-remix/templates/deno/Dockerfile:39]: ah beautiful

**Comment by jacob-ebey** [comment]
We have a deno deployment test to deno.deploy

**Pull request closed without merging**